index.js

```diff
@@ -551,6 +551,150 @@ studio.internal = (function(proto) {
     }
   }
 
+  obj.SystemNode = function(studioURL, notificationListener) {
+    var studioURL = studioURL;
+    var notificationListener = notificationListener;
+    var appConnections = new Array();
```

Comment:
Opening this new code in CDP Studio yields a lot of warnings. Should check and fix them.

![image](https://github.com/CDPTechnologies/JavascriptCDPClient/assets/7951792/6f29f35b-2e9d-44d7-bc9b-daded10e278a)